test/commands/lwc/test/setup.test.ts

```diff
@@ -359,11 +362,11 @@ describe('force:lightning:lwc:test:setup', () => {
     test
     .do(() => {
       stubMethod($$.SANDBOX, child_process, 'spawnSync').callsFake(cmd => {
-        if (cmd === 'node') {
-          return { status: 0, stdout: 'v8.12.0' };
+        if (cmd === 'node -v') {
```

Comment:
missed replacing `spawnSync` for this testcase I think, unless it was intentional.

Reply:
I found this issue, this is because when not mocked, the `spawnSync` method is invoked and executes the real command which returns the current node version installed on the machine.